server/utils_test.go

```diff
@@ -145,3 +146,31 @@ func TestCheckMinVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestSanitizeFilename(t *testing.T) {
+	tcs := []struct {
+		name     string
+		input    string
+		expected string
+	}{
+		{
+			name: "empty string",
+		},
+		{
+			name:     "spaces",
+			input:    "some file name with spaces.mp4",
+			expected: "some_file_name_with_spaces.mp4",
+		},
+		{
+			name:     "special chars",
+			input:    "somefile*with??special/\\chars.mp4",
+			expected: "somefile_with__special__chars.mp4",
+		},
```

Comment:
Should we test the too-long case also?

Reply:
Yep, added on the specific function, thanks :+1: